tests/many_to_many/tests.py

```diff
@@ -152,6 +152,20 @@ def test_fast_add_ignore_conflicts(self):
         with self.assertNumQueries(1):
             self.a1.publications.add(self.p1, self.p2)
 
+    @mock.patch('django.db.connection.features.supports_ignore_conflicts', False)
```

Comment:
This makes this test run on other backends then Oracle. It made the patch slightly easier to test locally but I don't have a strong opinion about whether or not it should be kept.

Reply:
I think we should skip this test on other backends, because only on Oracle we execute a SELECT query.